### ISSUE TITLE
[3.5.0 RC1] Not loading after upgrade

### ISSUE THREAD

**Issue by derekblankmccoy** [nextcloud/android#3539]
### Actual behaviour
- When upgrading the app (I tested upgrading from 3.2.1 which is the latest on download.nextcloud.com) files do not load unless previously cached.

### Expected behaviour
- Files should download and folders should populate
 
### Steps to reproduce
1.  Download and install a previous version, I tested 3.2.1 (https://download.nextcloud.com/android/nextcloud-30020199.apk)
2. Sign in and let it load the file/folder list
3. Upgrade to 3.5.0 RC1 and open the app
4. Files will not download and opening a folder will just give a spinner
5. Kill the app
6. Open the app again
7. App behaves normally

### Environment data
Android version: 7.0

Device model: Galaxy Tab A

Stock or customized system: No

Nextcloud app version: 3.2.1 -> 3.5.0C1

Nextcloud server version: 14.03

**Comment by tobiasKaminsky** [comment]
So this is a one time problem after upgrading?

**Comment by derekblankmccoy** [comment]
Yes, however it didn't used to happen before.

**Comment by tobiasKaminsky** [comment]
And it should not happen at all, but with a one time problem it is really hard to debug…
I tried it with and Android 7 emulator, to have the same version as you.
I used 3.2.1 as you and then 3.5.0 RC1 (both "nextcloud" not gplay) and it worked.

Sorry to say, but I doubt that i cannot really debug this further if it is not reproducible…
But thank you for your bug report and hopefully this is not happening to other users.
